docs/rules/order.md

```diff
@@ -132,6 +132,7 @@ Properties of the objects
 
 This defines import types that are not handled by configured pathGroups.
 This is mostly needed when you want to handle path groups that look like external imports.
+This option allows more than only `groups`(e.g., `builtin`, `external`, etc.), you can also use `patterns`(e.g., `react`, `react-router-dom`, etc).
```

Comment:
Maybe instead of the sentence, it'd be clearer to add a second example using `patterns`?

Reply:
It’s been added, please let me know if there is anything necessary to be revised.